locallib.php

```diff
@@ -2588,6 +2588,24 @@ function bigbluebuttonbn_settings_clienttype(&$renderer) {
     }
 }
 
+/**
+ * Helper function renders general settings if the feature is enabled.
+ *
+ * @param object $renderer
+ *
+ * @return void
+ */
+function bigbluebuttonbn_settings_muteonstart(&$renderer) {
+    // Configuration for BigBlueButton.
+    if ((boolean)\mod_bigbluebuttonbn\settings\validator::section_muteonstart_shown()) {
+        $renderer->render_group_header('muteonstart');
+        $renderer->render_group_element('muteonstart_default',
+            $renderer->render_group_element_checkbox('muteonstart_default', 0));
+        $renderer->render_group_element('muteonstart_editable',
+            $renderer->render_group_element_checkbox('muteonstart_editable', 1));
```

Comment:
muteonstart_editable should be 0 by default

This feature is for covering a very edge case. We don;t want to add unnecesary settings to the Teachers with the new version.